deepchem/models/lightning/dc_lightning_dataset_module.py

```diff
@@ -26,7 +26,7 @@ class DCLightningDatasetModule(pl.LightningDataModule):
     This class requires PyTorch to be installed.
     """
 
-    def __init__(self, dataset, batch_size, collate_fn):
+    def __init__(self, dataset, batch_size, collate_fn, num_workers: int):
```

Comment:
docs reminder

Reply:
Let's just add type annotations while we're at it here

Reply:
Added type annotations and docs